interface/modules/custom_modules/oe-module-visualehr/react/vehr/src/services/fetchRequest.jsx

```diff
@@ -0,0 +1,102 @@
+
+import { fromFetch } from "rxjs/fetch";
+import { switchMap, of, catchError } from "rxjs";
+import { rxService } from './Services';
+import { UtilService } from './UtilService';
+
+export const fetchService = {
```

Comment:
Ok, I'm sorry but I just don't see how your current implementation can be allowed inside of OpenEMR.  

You cannot use a third party site (medsov.com) to send created issues, problems, devices, etc to display and render on the timeline.  

1) This is a violation of US law w/ HIPAA without medsov having a BAA with every single installation of OpenEMR. There is no mechanism here in your plugin where you are offering a BAA for users to sign before using your plugin.
2) Our users trust OpenEMR to not transmit data to a third party w/o their consent, 
3) This doesn't even grab any of the data that is in the existing EMR installation.  I looked around the code but I don't see any calls to our FHIR or standard API to grab the existing customer data.  This forces users to only use the functionality in the VEHR to see data since its stored on the medsov servers.  Why even use OpenEMR if the data for patients is stored on the medsov server.

I hope I am horribly misunderstanding the code, but I've stepped through it with a network debugger and gone through the code as far as I can tell this is the case.

Reply:
Whatever happened to using the native FHIR or standard api?  I know I helped work through some of those issues with The medsov team's original iteration. Do you need to bring these php scripts (issues.php, fetch.phpactive_codes.php, icd10.php, etc) into your php module?  If you are able to bring these scripts into the module and hit the OpenEMR backend (not medsov.com) I can see that being a viable option.  Otherwise I don't see this happening.